Backend/UnitTesting/App.config

```diff
@@ -1,25 +1,40 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure why there is an App.config file being used, should already have an app.config file with this layout